docs/markdown/android/toast.md

```diff
@@ -0,0 +1,158 @@
+  </Group>
+  <Group>
+  <Dont title="When not to use" />
+  - When, due to an error, a user can’t even continue performing basic tasks like browsing already loaded Pins.
+  - When asking a user to confirm that they want to perform an action. Use [Sheet](/android/sheet) instead.
+  - When you want to suggest a user spend more money or try new features; use [Upsell](/upsell) instead.
+  - For errors that relate to a specific section or page. Use [Callout](/web/callout) or [SlimBanner](/slimbanner) instead.
+  - To guide or educate the user. Use [Popover](/popover) or [Tooltip](/tooltip) instead.
+  </Group>
+</TwoCol>
+
+## Best practices
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/69/d5/3f/69d53f0b5e459aeca189acba1949111a.jpg" alt="example with toast with unblocking placement"/>
+    <Do title="Do" />
+    Place Toasts out of the way so that a user can still navigate and complete tasks. Keep a bottom margin that is the same size as the left and right margins.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/8b/ae/9a/8bae9aff403a1a62baf18ea684eb8578.jpg" alt="example of with blocking placement"/>
+    <Dont title="Don't" />
+    Block navigation controls with Toasts or align too close to the edge of a navigation bar.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/b4/eb/3d/b4eb3d14e0997ca8cfc6495f951880e0.jpg" alt="example of showing one toast"/>
+    <Do title="Do" />
+    Show one Toast at a time, with errors and Acknowledgements taking priority.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/65/eb/4e/65eb4e3f85d7774955cfcd8f4d875d46.jpg" alt="example of showing multiple toasts"/>
+    <Dont title="Don't" />
+    Stack multiple toasts as that will block the user.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/5d/78/fc/5d78fc69a86c9c70311719f6842feaf4.jpg" alt="example of a dismissible toast"/>
+    <Do title="Do" />
+    Include a way to dismiss the toast when it is actionable or contains multiple lines of text. Mobile toasts can be dismissed via swiping on the toast in any direction.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/32/66/29/326629104e4376649ed9c910516423c2.jpg" alt="example of leaving a toast on screen"/>
+    <Dont title="Don't" />
+    Leave toasts on screen for a long time without a way to dismiss. Exceptions are blocking error toasts where a user can’t take any action until the error is resolved.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple’s accessibility features, such as reduced transparency, VoiceOver, and increased text size to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple’s Human Interface Guidelines and documentation about accessibility for iOS:
+
```

Comment:
this all needs to be updated for Android

Reply:
Done